src-ocaml/baijiu_md5.ml

```diff
@@ -202,27 +202,26 @@ module Make (B : Baijiu_buffer.S)
 
     ()
 
-  let feed_bytes = feed ~blit:B.blit_from_bytes ~le32_to_cpu:B.le32_from_bytes_to_cpu
-  let feed_bigstring = feed ~blit:B.blit_from_bigstring ~le32_to_cpu:B.le32_from_bigstring_to_cpu
-  let feed = feed ~blit:B.blit ~le32_to_cpu:B.le32_to_cpu
+  let unsafe_feed_bytes = feed ~blit:By.blit ~le32_to_cpu:By.le32_to_cpu
+  let unsafe_feed_bigstring = feed ~blit:By.blit_from_bigstring ~le32_to_cpu:Bi.le32_to_cpu
 
-  let get ctx =
-    let padding = B.create 64 in
-    let bits = B.create 8 in
-    let res = B.create (4 * 4) in
+  let unsafe_get ctx =
+    let padding = By.create 64 in
+    let bits = By.create 8 in
+    let res = By.create (4 * 4) in
 
-    B.set padding 0 '\x80';
-    B.fill padding 1 63 '\x00';
-    B.cpu_to_le64 bits 0 Int64.(ctx.size lsl 3);
+    By.set padding 0 '\x80';
+    By.fill padding 1 63 '\x00';
+    By.cpu_to_le64 bits 0 Int64.(ctx.size lsl 3);
 
     let index = Int64.(to_int (ctx.size land 0x3FL)) in
     let padlen = if index < 56 then 56 - index else (64 + 56) - index in
 
-    feed ctx padding 0 padlen;
-    feed ctx bits 0 8;
+    unsafe_feed_bytes ctx padding 0 padlen;
```

Comment:
if `padlen` can be less than 60, wouldn't it make sense to compute `padlen` first and allocate a smaller `padding` if we don't need all of it?